lib/mod/cache/mod.go

```diff
@@ -0,0 +1,14 @@
+package cache
+
+import "strings"
+
+func splitMod(mod string) (remote, owner, repo string) {
```

Comment:
@b4nst Thoughts on how we might deal with a `cuelang.org/go` type repository?

Reply:
Meaning one without the `owner` part? Maybe just returning empty owner will trick the rest of the code?

Reply:
I was thinking about if an empty owner would do it, but haven't looked at this code in a while 😕

Reply:
I'm gonna give it a look after cleaning some of my mess